src/ECER.Clients.RegistryPortal/ECER.Clients.RegistryPortal.Server/Applications/ApplicationsEndpoints.cs

```diff
@@ -79,6 +80,31 @@ public void Register(IEndpointRouteBuilder endpointRouteBuilder)
         .RequireAuthorization()
         .WithParameterValidation();
 
+
+    endpointRouteBuilder.MapGet("/api/applications/{id}/status", async Task<Results<Ok<SubmittedApplicationStatus>, BadRequest<ProblemDetails>, NotFound<ProblemDetails>>> (string id, HttpContext ctx, IMediator messageBus, IMapper mapper, CancellationToken ct) =>
+    {
+      var userId = ctx.User.GetUserContext()?.UserId;
+      bool IdIsNotGuid = !Guid.TryParse(id, out _); if (IdIsNotGuid)
```

Comment:
Not a syntax we generally use in C#. Make sure your auto formatting is enabled and that you've installed the right extensions